CI: Docs Sync fails on main HEAD — crates/mika-agent/docs/runtime-structure.md stale vs canonical

## Evidence

Last CI run on main HEAD (run 31024661195, push 08-05): **Docs Sync: failure**. Fails on every PR that touches nothing docs-related — confirmed on PR#1899 (γ ship, merged 08-05) and PR#1902 (word-boundary matcher, opened 08-16).

CI log excerpt (job 95231047879):
```
--- a/crates/mika-agent/docs/runtime-structure.md
+++ a/crates/mika-agent/docs/runtime-structure.md
@@ -43,7 +43,7 @@
-│       ├── identity.toml             # Agent name + emoji (0600)
+│       ├── identity.toml             # Agent name + emoji + [kg]/[skills]/[tools]/[context]/[session] blocks (0600)
...
-**sessions** — `id TEXT PK`, `agent_id TEXT FK→agents`, ...
+**sessions** — `id TEXT PK`, `agent_id TEXT FK→agents`, ... Session IDs are prefix-typed: random UUID (per-ask default), `system-{agent_id}` ...

##[error]Crate-local docs are out of sync with docs/
Run: bash scripts/sync-agent-docs.sh
Then commit the updated files in crates/mika-agent/docs/
```

## Root cause

Canonical `docs/runtime-structure.md` was updated (session prefix-types content — likely from mika#1401 session config work). Sync script `bash scripts/sync-agent-docs.sh` was not re-run to propagate to `crates/mika-agent/docs/runtime-structure.md`, so the crate-local copy is stale. CI checks parity between the two and rejects any drift.

## Impact

- Blocks CI-green on every mika PR (2/3 rouges sur PR#1902 diagnostic)
- Substrate health : main baseline rouge = pas de baseline vert pour valider PR régression vs pré-existant

## Fix

```bash
bash scripts/sync-agent-docs.sh
git add crates/mika-agent/docs/runtime-structure.md
git commit -m "docs: sync runtime-structure.md — session prefix-types + identity blocks"
```

## Priority

**p1-important** — blocker CI-green pour tous les PRs mika, pas de sécurité mais convention drift → PR reviewers doivent naviguer autour d'un rouge pré-existant à chaque merge.

## Filing context

Diagnostic 08-17 pendant investigation PR#1902 CI red. Ticket créé par MPC après hard-evidence collect (fail same run on #1899 + main HEAD).

--- a/crates/mika-agent/docs/runtime-structure.md
+++ b/crates/mika-agent/docs/runtime-structure.md
@@ -43,7 +43,7 @@ Home directory: `$MIKA_HOME` (default `~/.mika/`).
 ├── agents/
 │   └── mika/                         # Default agent (same structure per agent)
 │       ├── config.toml               # Per-agent config overrides (0600)
-│       ├── identity.toml             # Agent name + emoji (0600)
+│       ├── identity.toml             # Agent name + emoji + [kg]/[skills]/[tools]/[context]/[session] blocks (0600)
 │       ├── soul.md                   # Personality definition (0600)
 │       ├── heartbeat.md              # Heartbeat checklist (0600)
 │       ├── user.md                   # User info seed (0600)
@@ -114,7 +114,7 @@ Durable changes to bundled skills must go through the source tree at `skills/bun
 
 **agents** — `id TEXT PK`, `name TEXT NOCASE`, `home_dir TEXT`, `active BOOLEAN DEFAULT 1`, `last_seen TEXT`, `created_at TEXT`
 
-**sessions** — `id TEXT PK`, `agent_id TEXT FK→agents`, `channel_type TEXT DEFAULT 'cli'`, `parent_session_id TEXT`, `task_id TEXT` (reverse link to triggering task, partial index), `started_at TEXT`, `ended_at TEXT`, `metadata TEXT`
+**sessions** — `id TEXT PK`, `agent_id TEXT FK→agents`, `channel_type TEXT DEFAULT 'cli'`, `parent_session_id TEXT`, `task_id TEXT` (reverse link to triggering task, partial index), `started_at TEXT`, `ended_at TEXT`, `metadata TEXT`. Session IDs are prefix-typed: random UUID (per-ask default), `system-{agent_id}` (compaction summaries), `canonical-{agent_id}` (single-session-by-nature agents, mika#1401), plus derived namespaces `heartbeat-*`, `callback-*`, `skill-*`, `reflection-*`, `team-*`, `delegate-*`, `reminder-*`. `prune_old_sessions` only deletes the derived-namespace prefixes; `canonical-*`, `system-*`, and bare UUIDs are exempt. Agents opt into a single canonical session via `[session] singleton = true` in `identity.toml` (see `crates/mika-agent/CLAUDE.md` § Session Configuration).
 
 **messages** — `id INTEGER PK AUTO`, `session_id TEXT FK→sessions`, `agent_id TEXT FK→agents`, `role TEXT CHECK(user|assistant|system|summary|tool_result)`, `content TEXT`, `metadata TEXT`, `trace_id TEXT`, `compacted_through_id INTEGER`, `created_at TEXT`, `internal INTEGER NOT NULL DEFAULT 0` (agent-to-agent messages hidden from TUI inbox mode)
 

--- a/deny.toml
+++ b/deny.toml
@@ -6,8 +6,16 @@ db-path = "~/.cargo/advisory-db"
 db-urls = ["https://github.com/rustsec/advisory-db"]
 
 [[advisories.ignore]]
-id = "RUSTSEC-2024-0436"
-reason = "paste is unmaintained but has no security impact; widely used transitive dependency with no drop-in replacement yet."
+id = "RUSTSEC-2026-0253"
+reason = "lru LruCache::pop panic-safety (unsound); transitive dep only, we do not pass panicking Drop keys to pop()."
+
+[[advisories.ignore]]
+id = "RUSTSEC-2026-0221"
+reason = "event-listener !Send tags via StackSlot (unsound); transitive dep only, we do not share listeners across threads on custom !Send types."
+
+[[advisories.ignore]]
+id = "RUSTSEC-2026-0205"
+reason = "scc Array::insert exception safety (unsound); transitive dep only, we do not pass panicking compare fns."
 
 [licenses]
 allow = [

